internal/common/conversion/schema_description.go

```diff
@@ -0,0 +1,73 @@
+package conversion
```

Comment:
curious if we could unit test this, but might not be worth it?

Reply:
we might, but TestResourceSchemas is testing it indirectly

Reply:
@EspenAlbert  at the end I created some unit tests here https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2605/commits/318c46cfab21a43b42bd444a66a6573dcb359a59